04 Abstraction Exercises Part 1/abstraction-chat-filtering-project/src/main/java/Chat.java

```diff
@@ -0,0 +1,47 @@
+ /**
+ * This class is the Chat class
+ * */
+
+import java.util.ArrayList;
+        import java.util.List;
+
+public class Chat {
+    private String senderName;
+    private String recipientName;
+    private List<String> messages;
+    private boolean recipientOnline;
+    private List<String> bannedWords;
```

Comment:
this should be used in all the instances of the Chat class

Reply:
Noted sir